app/services/configs/course_information.rb

```diff
@@ -0,0 +1,25 @@
+module Configs
```

Comment:
If you created a class for this, why use the `yml` file? You can have the values in a hash here. Also, is there a requirement for this to be a "configuration"? To me it doesn't feel like something that should be in `/config`

Reply:
I don't have an issue with storing data in yaml files, but I'm not a fan of handling it as a configuration. You are calling it `static data` in your comment as well.

Does it not look a bit off having `Configs` here?:

```ruby
@course_information ||= Configs::CourseInformation.new(@course)
```

The way I'd see this working is something similar to the `ActiveYaml` implementation of [active_hash](https://github.com/active-hash/active_hash), potentially having a model for course information.

Reply:
Thoughts on adding `data` in the namespace? For example `configs/data/course_information.yml`?